pages/courses/[id].js

```diff
@@ -81,10 +85,20 @@ export async function getStaticPaths() {
 
 export async function getStaticProps({ params }) {
   const course = courses.find((c) => c.course_number === params.id)
+  const schoolsWithCourseNumbers = schoolsWhereCourseNumber(
+    course.course_number
+  )
+  const schools = schoolsWithCourseNumbers.map((school) => {
+    // Remove course_numbers because they aren't needed in this context
+    // TODO: why is course.course_number needed in the array?
```

Comment:
I think it's because of      filtering on `school.course_numbers.includes(course_number)` in the `schoolsWithCourseNumbers` function (see lib/models/school.js).